addons/binding/pom.xml

```diff
@@ -16,7 +16,7 @@
   <packaging>pom</packaging>
 
   <modules>
-    <module>org.openhab.binding.airquality</module>
+    <!--module>org.openhab.binding.airquality</module>
```

Comment:
Please unbreak this :-) and add it at the correct position

Reply:
I have reactivated all bindings and added my binding at the correct position according to alphabetical sorting